es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-light">
+                    1
+                </b-col>
+                <b-col
+                    cols="1"
+                    class="bg-gray-dark">
+                    1
+                </b-col>
+            </b-row>
+        </b-container>
+        <ds-doc-source
+            :doc-code="docCode"
+            doc-source="es-ds-docs/atoms/layout.vue" />
     </div>
 </template>
+<script setup lang="ts">
+import sassBreakpoints from '@energysage/es-bs-base/scss/modules/breakpoints.module.scss';
+import sassMaxWidths from '@energysage/es-bs-base/scss/modules/max-widths.module.scss';
+
+const  breakpointTableFields = [
+                { key: 'label', label: `` },
+                { key: 'xs', label: 'Extra small (xs)' },
+                { key: 'sm', label: 'Small (sm)' },
+                { key: 'md', label: 'Medium (md)' },
+                { key: 'lg', label: 'Large (lg)' },
+                { key: 'xl', label: 'Extra large (xl)' },
+                { key: 'xxl', label: 'Extra extra large (xxl)' }];
+const breakpointTableItems = [{
+                    label: 'Breakpoint',
+                    xs: `< ${sassBreakpoints.sm}`,
+                    sm: `≥ ${sassBreakpoints.sm}`,
+                    md: `≥ ${sassBreakpoints.md}`,
+                    lg: `≥ ${sassBreakpoints.lg}`,
+                    xl: `≥ ${sassBreakpoints.xl}`,
+                    xxl: `≥ ${sassBreakpoints.xxl}`,
+                },
+                {
+                    label: 'Max container width',
+                    xs: 'None',
+                    sm: sassMaxWidths.sm,
+                    md: sassMaxWidths.md,
+                    lg: sassMaxWidths.lg,
+                    xl: sassMaxWidths.xl,
+                    xxl: sassMaxWidths.xxl,
+                }
+            ];
+
+const { $prism } = useNuxtApp();
+const docCode = ref('');
+
+if ($prism) {
+    /* eslint-disable import/no-webpack-loader-syntax, import/no-self-import */
+    const docSource = await import('./layout.vue?raw');
+    /* eslint-enable import/no-webpack-loader-syntax, import/no-self-import */
+    docCode.value = $prism.normalizeCode(docSource.default);
+    $prism.highlight();
+}
+</script>
+
+
+<style lang="scss" scoped>
+@use "@energysage/es-bs-base/scss/variables" as variables;
+
+.bg-gray-light {
+    background: variables.$gray-400;
+    text-align: center;
+}
+
+.bg-gray-dark {
+    background: variables.$gray-700;
+    text-align: center;
+    color: variables.$gray-400;
```

Comment:
This is how it was before (for some reason), but we have proper utility classes for all of the custom CSS here. Can we replace all usages of `bg-gray-light` with `bg-gray-400 text-center`, and all usages of `bg-gray-dark` with `bg-gray-700 text-center text-gray-400` to model proper utility class usage?

Reply:
Yes, I removed the style section!